macos/Sources/Features/Command Palette/TerminalCommandPalette.swift

```diff
@@ -154,7 +163,7 @@ struct TerminalCommandPaletteView: View {
                 }
 
                 return CommandOption(
-                    title: "Focus: \(displayTitle)",
+                    title: isJumpPalette ? displayTitle : "Focus: \(displayTitle)",
```

Comment:
Remove this completely and change it so that in command palette mode we run a `map` over it which modifies the title to add a prefix.